app/controllers/reviews_controller.rb

```diff
@@ -3,6 +3,9 @@ def new
   end
 
   def edit
-    @review = Review.find(params[:park_id])
+    @review = Review.find(params[:id])
+  end
+
+  def destroy
   end
```

Comment:
If you're not using this action as part of this PR (seems like it's not doing anything currently), please remove it from this branch!